massa-execution-worker/src/tests/scenarios_mandatories.rs

```diff
@@ -321,6 +321,98 @@ fn send_and_receive_async_message() {
     manager.stop();
 }
 
+#[test]
```

Comment:
I think it's cool if we always have the walkthrough like in the others tests !

Reply:
Done